docker-compose.yml

```diff
@@ -11,6 +11,10 @@ services:
     build:
       context: .
       dockerfile: docker/cami-rabbitmq/Dockerfile
+    ports:
+      # Make the RabbitMQ server available on the host so VMs can access it
```

Comment:
@iprunache this can also be achieved by linking the machines together. Docker puts the port value and machine IP in environment variables in the container with which the port is being shared.